proxy.js

```diff
@@ -40,5 +45,26 @@ http.createServer(function (req, res) {
   if (process.getuid && process.setuid && process.env.SUDO_USER && process.getuid() === 0) {
     process.setuid(process.env.SUDO_USER);
   }
-
 });
+
+if (sslCert && sslKey) {
+  https.createServer({
+      key: fs.readFileSync(sslKey),
+      cert: fs.readFileSync(sslCert),
+      requestCert: false,
+      rejectUnauthorized: true
+    },
+    function (req, res) {
```

Comment:
This function should probably be extracted and named, since it's the same for both cases.